scripts/software/apps.sh

```diff
@@ -1,40 +1,243 @@
+# Fix Previous Command.
+check "thefuck" "--version"
+# Hex Editor.
+check "ghex" "--version"
+# GUI Diff.
+check "meld" "--version"
+# Code Indexing.
+check "ctags" "--version"
+# Code Search (a specialized grep).
+check "ack" "--version"
+
+'''
+Terminal Management.
+'''
+
+# Terminal Multiplexer.
+check "byobu" "--version"
+# Another Terminal Multiplexer.
+check "tmux" "--version"
+# Multiple Terminals Management.
+check "terminator" "--version"
+
+'''
+# TODO:
+'''
+
+# Executes Commands in Makefile.
 check "make" "--version"
+
+'''
+Version Control.
+'''
+
 check "git" "--version"
+check "subversion" "--version"
+
+'''
+GNOME.
+'''
 
 check "gnome-software" "--version"
 
+'''
+KDE.
+'''
+
+check "kde-applications" "--version"
+# Bulk rename.
+check "krename" "--version"
+# File-version control.
+check "kdiff" "--version"
+
+'''
+Programming Languages & Compilers.
+'''
+
+# GNU Compiler Collection (C, C++, Go...).
 check "gcc" "--version"
 check "java" "-version"
 check "python" "-V"
+# Haskell Compiler.
+check "ghci" "--version"
 check "ruby" "-v"
+# Ruby version control.
+check "rvm" "--version"
+# Another Ruby Version Manager.
+check "rbenv" "--version"
 check "perl" "-v"
+check "lua" "-v"
 check "mysql" "--version"
+
+'''
+PenTest.
+'''
+
 check "freedom" "-version"
+# Security Scanner.
 check "nmap" "-V"
+# Wire protocols inspector.
+check "wireshark-cli" "--version"
+
+'''
+Remote Connectivity.
+'''
+
+check "ssh" "-V"
+# Uses FUSE to mount folders remotely using SSH.
+check "sshfs" "--version"
+# FTP Client.
+check "filezilla" "--version"
+# Cross-Platform Multiple Computers Control.
+check "synergy" "--version"
+# Sync Dir-Trees in Multiple Systems.
+check "unison" "--version"
+# Dropbox Alternative.
+check "sparkleshare" "--version"
+
+'''
+Encryption.
+'''
+
+check "veracrypt" "--version"
+check "pgp" "--version"
+check "gpg" "--version"
+
+''' 
+Text Editors/IDEs.
+'''
+
 check "nano" "--version"
-check "vi" "--version"
+if hash vi --version 2>/dev/null; then
+    check "vi" "--version"
+else
+    check "vim" "--version"
+fi
 check "emacs" "--version"
+check "code" "--version"
+check "atom" "--version"
+check "sublime" "--version"
+check "netbeans" "--version"
+check "eclipse" "--version"
+check "kate" "--version"
+check "codelite" "--version"
+check "gedit" "--version"
+check "geany" "--version"
+check "kwrite" "--version"
+check "scite" "--version"
+
+'''
+Virtual Enviroment Testing.
+'''
+
+check "vagrant" "--version"
+check "docker" "--version"
+
+'''
+File Editors.
+'''
+
+# Text-Based File Manager.
+check "ranger" "--version"
+# Office Suite.
 check "libreoffice" "--version"
+# PDF Viewer.
+check "okular" "--version"
+# Another PDF Viewer.
+check "zathura" "--version"
+# Typesetting System.
+check "tex" "--version"
+# Diagrams Creator.
+check "dia" "--version"
+# Plots Creator.
+check "gnuplot" "--version"
+# 3D CAD Modeler.
+check "freecad" "--version"
+
+'''
+Window Managers.
+'''
+
+check "i3" "--version"
+
+'''
+# TODO:
```

Comment:
parted: Add misc/unsorted section.
htop: System Monitors
stracer: System Monitors
transmission-cli: Personally I group these with my browsers. Could also go to misc/unsorted section.

Reply:
:heavy_check_mark: